### PR TITLE
Improvements to FieldHelpers

### LINKED ISSUE #306: Improvements to FieldHelpers
In some situations it would very helpful to be able to use `FieldHelpers` with an `FftField` instead of only being able to use it with `PrimeField`

Also ongoing work on foreign field multiplication and turshi needs a flexible to/from bits interface.

### PR DESCRIPTION
**Pull request by jspada**
* Make FieldHelpers work on FftField
    * Added to/from bits interface
    * Extend and improve unit tests

Closes #306

### CODE REVIEW

**Review by querolita — APPROVED** [review]
Very neat and useful

File: utils/src/field_helpers.rs [hunk 1]
```diff
@@ -85,45 +110,97 @@ mod tests {
                 .is_ok(),
             true
         );
+
+        assert_eq!(
+            BaseField::to_hex(
+                BaseField::from_hex(
+                    "2eaedae42a7461d5952d27b97ecad068b698ebb94e8a0e4c45388bb613de7e08"
```
Comment (mrmr1993) [inline utils/src/field_helpers.rs:117]: Nit: factor out as a constant instead of repeating

File: utils/src/field_helpers.rs [hunk 2]
```diff
@@ -85,45 +110,97 @@ mod tests {
...
+            BaseField::from_bits(&fe.to_bits()).expect("Failed to deserialize field bits");
+        assert_eq!(fe, fe_check);
+
         assert_eq!(
-            ScalarField::from_hex(
-                "g64244176fddb5d769b7de2027469d027ad428fadcf0c02396e6280142efb7d8"
-            ),
-            Err("Failed to decode field hex")
+            BaseField::from_bits(&[
+                false, true, false, false, false, true, true, false, true, true, false, false,
```
Comment (mrmr1993) [inline utils/src/field_helpers.rs:165]: Can we convert the hex to bits in another fashion, to avoid hard-coding this?

**Review by mrmr1993 — APPROVED** [review]

File: utils/src/field_helpers.rs (hunk 2 above)
Comment (jspada) [reply utils/src/field_helpers.rs]: yes

**Review by mrmr1993 — APPROVED** [review]

**Pull request merged**